classes/item.rb

```diff
@@ -0,0 +1,36 @@
+require 'date'
+class Item
+  attr_accessor :publish_date, :genre, :author, :source, :label
+
+  def initialize(id, publish_date)
+    @id = id
```

Comment:
Could you please turn the Id into a Random and let the id in the parameter have a nil value  example `(id: nil) ``@id = id || Random.rand(1..1000)`

Reply:
issue resolved